### ISSUE TITLE
[QuickView] QuickView cannot be constructed

### ISSUE THREAD

**Issue by zLucPlayZ** [qwerasd205/BDPlugins#8]
Upon start of discord, I got the message "QuickView cannot be constructed". When clicking on the error modal in discord, it crashes.

I still pulled the error from the console tho.

```
Uncaught TypeError: Cannot read property 'bind' of undefined
    at VM721 QuickView.plugin.js:14
    at VM721 QuickView.plugin.js:191
    at Object.getFileModification (<anonymous>:4:307660)
    at Object..js (<anonymous>:4:267804)
    at Function.o._load (<anonymous>:4:370428)
    at Object.loadAddon (<anonymous>:4:267961)
    at Object.loadAddon (<anonymous>:4:305909)
    at FSWatcher.<anonymous> (<anonymous>:4:266207)
    ```

**Comment by YellowInkling** [comment]
Yeah I'm having the same issue too

**Comment by qwerasd205** [comment]
Fixed in `0.1.3`, just published - will take a bit to be verified on the website, but if you're impatient you can [download it directly](https://raw.githubusercontent.com/qwerasd205/BDPlugins/main/QuickView/QuickView.plugin.js).
